packages/components/accordion/src/accordion-item.ts

```diff
@@ -0,0 +1,119 @@
+import { localized } from '@lit/localize';
+import { type EventEmitter, event } from '@sl-design-system/shared';
+import { type CSSResultGroup, LitElement, type TemplateResult, html } from 'lit';
+import { property, query } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+import styles from './accordion-item.scss.js';
+
+let nextUniqueId = 0;
+
+/**
+ * An accordion item component.
+ *
+ * @slot default - Body content for the accordion
+ * @part summary - Header element of the accordion-item
+ */
+@localized()
+export class AccordionItem extends LitElement {
+  /** @private */
+  static override styles: CSSResultGroup = styles;
+
+  /**
+   * Unique ID for each accordion item component.
+   */
+  #accordionItemId = `sl-accordion-item-${nextUniqueId++}`;
+
+  /**
+   * Unique ID for each accordion item component content.
+   */
+  #accordionItemContentId = `sl-accordion-item-content-${nextUniqueId++}`;
+
+  /** A text shown in the header - as a title of the accordion item. */
+  @property() summary!: string;
+
+  /** Whether the element is disabled. */
+  @property({ type: Boolean, reflect: true }) disabled?: boolean;
+
+  /** Emits when the accordion item has been toggled. */
+  @event({ name: 'sl-toggle' }) toggleEvent!: EventEmitter<string>;
+
+  /** Whether the details element is opened. */
+  @property({ reflect: true, type: Boolean }) open?: boolean;
+
+  /** @private */
+  @query('.wrapper') wrapper!: HTMLDivElement;
```

Comment:
Since we only use this variable in the `#onClick` method, it might be better to just do `const wrapper = this.renderRoot.querySelector('.wrapper')` at the top of the `#onClick` method?